manifests/params.pp

```diff
@@ -4,6 +4,9 @@
 class candlepin::params {
   $ssl_port = 8443
 
+  $candlepin_version = 'latest'
```

Comment:
`container_version` perhaps? we already have a version but is probably the one you intended for the container version